package.json

```diff
@@ -63,6 +63,7 @@
     "passport-github2": "0.1.12",
     "passport-google-oauth20": "2.0.0",
     "passport-linkedin-oauth2": "^2.0.0",
+    "passport-spotify": "^1.1.0",
```

Comment:
Let's update this to https://github.com/JMPerez/passport-spotify/releases/tag/v2.0.0

Reply:
This is done